Prace_domowe/Praca_domowa1/Grupa4/yevhenii_vinichenko/pd_01.ipynb

```diff
@@ -0,0 +1,706 @@
+{
```

Comment:
<p>ok</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/10/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
<p>Zamiast total area, można się pokusić o średnią&nbsp;i odchylenie, bo to jest więcej info o rozkładzie</p>